tgui/packages/tgui/interfaces/BigManipulator.tsx

```diff
@@ -33,6 +32,82 @@ type PrioritySettings = {
   priority_width: number;
 };
 
+const DelayControls = ({ act, data }) => {
```

Comment:
Pls do not pass act and data as props, they're available globally with the usebackend hook